apps/nextjs/package.json

```diff
@@ -12,26 +12,26 @@
   "dependencies": {
     "@acme/api": "*",
     "@acme/db": "*",
-    "@tanstack/react-query": "^4.7.2",
-    "@trpc/client": "10.0.0-proxy-beta.13",
-    "@trpc/next": "10.0.0-proxy-beta.13",
-    "@trpc/react": "10.0.0-proxy-beta.13",
-    "@trpc/server": "10.0.0-proxy-beta.13",
-    "next": "12.3.1",
-    "react": "18.1.0",
-    "react-dom": "18.1.0",
+    "@tanstack/react-query": "^4.3.8",
+    "@trpc/client": "10.0.0-rc.3",
+    "@trpc/next": "10.0.0-rc.3",
+    "@trpc/react-query": "10.0.0-rc.3",
+    "@trpc/server": "10.0.0-rc.3",
+    "next": "13.0.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
     "zod": "^3.18.0"
   },
   "devDependencies": {
     "@types/node": "^18.0.0",
-    "@types/react": "^18.0.21",
-    "@types/react-dom": "^18.0.5",
+    "@types/react": "^18.0.24",
+    "@types/react-dom": "^18.0.8",
     "@typescript-eslint/eslint-plugin": "^5.33.0",
     "@typescript-eslint/parser": "^5.33.0",
     "autoprefixer": "^10.4.7",
     "eslint": "8.22.0",
-    "eslint-config-next": "12.3.1",
-    "next-transpile-modules": "^9.0.0",
+    "eslint-config-next": "13.0.1",
+    "next-transpile-modules": "^10.0.0",
```

Comment:
Next has (experimental), built in transpiling in 13. Should we try it? 

https://github.com/vercel/next.js/blob/3cb0c297c89fde815a34f23528ea41d211d57e3c/packages/next/server/config-shared.ts#L163